src/main/java/testutils/TabsUtil.java

```diff
@@ -0,0 +1,24 @@
+package testutils;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import pages.WindowPage;
+
+public class TabsUtil {
+    private static String testResult = "New page title: %s\nNew page url: %s\nNew page text: %s";
+
+    public static void getNewPageData(WebDriver driver, WindowPage windowPage, String parentPage) {
```

Comment:
I see "get" in name of the method, but don't see what method returns. Actually, I don't understand what method does at all. Let's explain me on the call